shell/platform/android/io/flutter/embedding/android/AndroidKeyProcessor.java

```diff
@@ -154,12 +37,7 @@ public boolean isPendingEvent(@NonNull KeyEvent event) {
    * <p>The following reference explains the concept of a "combining character":
    * https://en.wikipedia.org/wiki/Combining_character
    */
-  @Nullable
-  private Character applyCombiningCharacterToBaseCharacter(int newCharacterCodePoint) {
-    if (newCharacterCodePoint == 0) {
```

Comment:
Is there a reason that we treated `0` differently and returned `null` specifically in this case (and didn't clear the combining key).

Reply:
If so, I don't remember it.  Maybe zero is a special code point that indicates some change in state, or informational event that should be ignored? It could also be that zero indicates a non-printable character like shift or meta, in which case we might ignore it since it might allow for combining with an uppercase character or something. Anyhow, seems like we should look into why we might get a zero here.